docs/adr/adr-0017-finding-accessible-colors-and-creating-design-tokens.md

```diff
@@ -0,0 +1,123 @@
+
+#### Tools
+To find these colors, we relied on the [Color Review](https://color.review/check/FEDC2A-5A3B5D) tool that we found thanks to [Stéphanie Walter's blog that lists many other great tools](https://stephaniewalter.design/blog/color-accessibility-tools-resources-to-design-inclusive-products/).
+
+#### Number of colors
+We chose to only keep 2 shades of our main colors (blue, green, orange and red):
+* the main color (to be used as a text color or a plain background that would make it stand out from the rest),
+* a very light shade of the color (to be used for background only, to make a text stand out but not too much).
+
+As for the grey color, we needed at least 10 shades since it is used in many different cases (2 shades for text, 2 shades for background, and shades for the differents states of the background like disabled, active, hovered, etc.).
+
+We know we may need to **add more shades or colors later on but we want to only select and add them when needed**.
+
+### Create Design Tokens
+The Design Token concept comes from the Salesforce team and has been defined by them as follows:
+> Design tokens are the visual design atoms of the design system — specifically, they are named entities that store visual design attributes. We use them in place of hard-coded values (such as hex values for color or pixel values for spacing) in order to maintain a scalable and consistent visual system for UI development.
+
+Source: [salesforce - Design Tokens](https://www.lightningdesignsystem.com/design-tokens/).
+
+Our work on Design Tokens is based on two articles:
+* [Design Tokens - What are they & how will they help you? by Lukas Oppermann](https://lukasoppermann.medium.com/design-tokens-what-are-they-how-will-they-help-you-b73f80f602ab)
+* [Naming Tokens in Design System by Nathan Curtis](https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676)
+
+These articles helped us grasp the concept of Design Tokens and find an architecture / strategy that would fit our needs.
+
+Note: Color Design Tokens are only the first step, see "What's next" section.
+
+#### No abstraction level
+Most of the examples we came accross insisted on building Design Tokens that were **Technology Agnostic and relied on tools to transform tokens into variables** that could be consumed with each specific technology (Web, Native mobile platforms, etc.).
+
+This made sense for very big project distributed accross several different platforms but our project is a little bit different:
+* People working on this project all are developers,
+* Our components are only for the Web and as close as possible to the standards.
+
+For these reasons, we chose to **only rely on CSS variables and avoid using additional tools** like [Style Dictionary](https://amzn.github.io/style-dictionary/#/) that would only have added more complexity with no actual benefits.
```

Comment:
We could add that changing this decision would not be difficult at all.

Reply:
does the sentence below seems ok to you or do you want me to elaborate more on that ?

Reply:
Perfect